Makefile

```diff
@@ -1,7 +1,11 @@
 BRANCH = "master"
 REPONAME = "neo-go"
 NETMODE ?= "privnet"
-BINARY = "./bin/neo-go"
+EXTENSION=
+ifeq ($(shell go env GOOS),windows)
```

Comment:
This won't work when GOOS is provided via CLI arguments: `make GOOS=windows`. Though I am not sure if anyone use it like this at all and `GOOS=windows make` works anyway.

Reply:
Well, the most fearest thing with it is that the resulting binary will have no extension. I think if user builds it with `make GOOS=windows` then he can add the proper extension by itself.

Reply:
We can do something like `GOOS ?= $(shell go env GOOS)` and check for `$(GOOS)` which should work for all cases.

Reply:
Nice solution, implemented.